Tools/Scripts/libraries/webkitscmpy/webkitscmpy/program/conflict.py

```diff
@@ -22,13 +22,16 @@
 
 import sys
 
+from webkitbugspy import Tracker, radar
 from .command import Command
 from .. import local
+from ..commit import Commit
 
 
 class Conflict(Command):
     name = 'conflict'
     help = "Given the representative Radar ID of a conflicting merge, checkout the branch."
+    INTEGRATION_BRANCH_PREFIX = 'integration'
 
     @classmethod
     def parser(cls, parser, loggers=None):
```

Comment:
It might be a quality of life improvement to change `radar_id` to a string, so you can input anything in the form of `rdar://problem/123456789`, `rdar://123456789`, etc.

Reply:
updated!